tests/DoctrineMongoODMModuleTest/Doctrine/ConfigurationFactoryTest.php

```diff
@@ -73,30 +61,31 @@ public function testCreation() : void
                             'metadata_cache' => 'stubbed_metadatacache',
-                            'driver' => 'stubbed_driver',
+                            'driver'         => 'stubbed_driver',
 
-                            'generate_proxies' => $proxyGenerate = false,
-                            'proxy_dir' => $proxyDir             = 'dir/proxy',
-                            'proxy_namespace' => $proxyNamespace = 'ns\proxy',
+                            'generate_proxies' => $proxyGenerate = Configuration::AUTOGENERATE_EVAL,
+                            'proxy_dir'        => $proxyDir = 'dir/proxy',
+                            'proxy_namespace'  => $proxyNamespace = 'ns\proxy',
 
-                            'generate_hydrators' => $hydratorGenerate  = true,
-                            'hydrator_dir' => $hydratorDir             = 'dir/hydrator',
-                            'hydrator_namespace' => $hydratorNamespace = 'ns\hydrator',
+                            'generate_hydrators'              => $hydratorGenerate = Configuration::AUTOGENERATE_ALWAYS,
+                            'hydrator_dir'                    => $hydratorDir = 'dir/hydrator',
+                            'hydrator_namespace'              => $hydratorNamespace = 'ns\hydrator',
                             // phpcs:disable Generic.Files.LineLength
                             'generate_persistent_collections' => $collectionGenerate = Configuration::AUTOGENERATE_EVAL,
                             // phpcs:enable Generic.Files.LineLength
-                            'persistent_collection_dir' => $collectionDir             = 'dir/collection',
+                            'persistent_collection_dir'       => $collectionDir = 'dir/collection',
                             'persistent_collection_namespace' => $collectionNamespace = 'ns\collection',
-                            'persistent_collection_factory' => PersistentCollectionFactory::class,
+                            'persistent_collection_factory'   => PersistentCollectionFactory::class,
                             'persistent_collection_generator' => PersistentCollectionGenerator::class,
 
-                            'default_db' => 'default_db',
-                            'filters' => [], // ['filterName' => 'BSON\Filter\Class']
-                            'types' => [$typeName = 'foo_type' => $typeClassName = CustomType::class],
-                            'classMetadataFactoryName' => 'stdClass',
-                            'repositoryFactory' => CustomRepositoryFactory::class,
+                            'default_db'                             => 'default_db',
+                            'filters'                                => [], // ['filterName' => 'BSON\Filter\Class']
+                            'types'                                  => [
+                                $typeName = 'foo_type' => $typeClassName = CustomType::class],
```

Comment:
Move the closing bracket onto its own line instead of appended to the assignment

Reply:
fixed.
I already have a [branch with the current doctrine coding standard](https://github.com/rieschl/DoctrineMongoODMModule/compare/odm-2...rieschl:doctrine-coding-standard) in place, but it's based off this PR, so I'll wait until this one is merged. It was straight forward, mostly spacing issues. Everything could be fixed by `phpcbf`.